.gitignore

```diff
@@ -2,3 +2,4 @@
 /node_modules
 /npm-debug.log
 .DS_Store
+.idea
```

Comment:
This folder is used by JetBrains (WebStorm, PyCharm and etc.) products to store its service files